packages/web-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,42 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTAccordionItem } from '@microsoft/fast-foundation';
+import { AccordionItemExpandIconPosition, AccordionItemSize } from './accordion-item.options.js';
+
+/**
+ * @internal
+ */
+class AccordionItem extends FASTAccordionItem {
```

Comment:
I think we tend to export from here when there is a single class in the file:

```suggestion
export class AccordionItem extends FASTAccordionItem {
```